### PR TITLE
Removes Flashing instances in game

### PR DESCRIPTION
**Pull request by pepperpow**
Accessibility (of the human variety) of removing instance of full screen flashing for players and stream viewers:

Disables the following (and should be tested for any speed inconsistency before PR):

- Aga Lightning Attack
- Aga Cutscene (pre-aga1, introduction demo)
- Vitreous lightning (needs to be checked)
- Weather effects (in standard intro and misery mire)
- Ether (flashing effects)
- Mad Batter

### CODE REVIEW

File: AdjusterMain.py [hunk 1]
```diff
@@ -28,9 +28,10 @@ def adjust(args):
     palettes_options['sword']=args.sword_palettes
     palettes_options['shield']=args.shield_palettes
     # palettes_options['link']=args.link_palettesvera
+    racerom = rom.read_byte(0x180213) > 0
```
Comment (Berserker66) [inline AdjusterMain.py:31]: This line doesn't appear to be used.

**Review by Berserker66 — CHANGES_REQUESTED** [review]

File: AdjusterMain.py (hunk 1 above)
Comment (pepperpow) [reply AdjusterMain.py:31]: Now it is.

**Comment by pepperpow** [comment]
Oh, latest commit added TWO new features: anti flashing on Dark Mountain and tile reveals on Ether

Separate ASM pull on z3randomizer for a fix on Mountain and Misery Mire for corrupted rain graphics

**Comment by Berserker66** [comment]
Thank you!

**Pull request merged**